content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+
+### Create a dedicated connection
+
+First, in order to communicate with your connector and obtain API credentials, Julia needs to create a dedicated [connection](https://help.akeneo.com/pim/serenity/articles/manage-your-connections.html) for your connector.
+
+To do so:
+1. Go to the `System` menu and then, on the `Connections` submenu
+2. Click on the `Create` button
+3. In the `Label` field, enter the name of your connector.
+4. Choose the `Other` flow type
+
+Whenever you create a connection, the PIM automatically generates a set of credentials for you.
+
+These API credentials consist of 4 different information:
+* the `client id`,
+* the `secret`,
+* the connection `username`,
+* the connection `password`.
+
+:::warning
+Don't forget to set the [permissions](https://help.akeneo.com/pim/serenity/articles/manage-your-connections.html#set-the-permissions) of this connection!
+:::
+
+As you may understood, your connector need to have a dedicated UI or a configuration file in order to manage these PIM API credentials.
+
+### Check Julia's PIM data
+
+Then, in order for your connector to be able to add translations, Julia's PIM must be correctly set up for that.
+
+:::warning
+That's why you must check with Julia that in her PIM:
+* All locales required for translations have been activated.
+* All attributes requiring translation for products or product models are set to "localizable".
+:::
+
+### Create a Translation attribute group
+
+First, in order to not mix these new product attributes with other Julia's product attributes, it is necessary that she creates **a specific attribute group** named `"Translations"`.
+
+To create this attribute group:
+1. Go to `Settings > Attribute groups`
+2. Click on `CREATE` at top right
+3. Add a new attribute group with the code: `Translations`.
+
+
+### Create Translation attributes
+
+Then Julia must create 6 new attributes in this attribute group:
+
+* `Translation submission name`
+* `Translation submitter`
+* `Translation queued`
+* `Translation locales`
+* `Translation due date`
+* `Translation status`
+
+:::tips
+To create an attribute:
+1. Go to `Settings > Attributes`
+2. Click on `CREATE ATTRIBUTE` at top right
+3. Choose the correct attribute type
+4. Then set the below information
+:::
+
+
+`Translation submission name`  
+**Attribute type:** Text  
+**Attribute code:** translationName  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This attribute allows Julia to set a name to her translation project.  
+It also allows her to easily find this project with this name in the interface of your online Translation solution.
+
+`Translation submitter`  
+**Attribute type:** Simple select  
+**Attribute code:** translationSubmitter  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This simple select attribute should contains in its `options` all possible contributor names. This attribute allows Julia to associate her name to her translation project.
+
```

Comment:
```suggestion
* `Translation submission name`: This attribute allows Julia to set a name to her translation project. It also allows her to easily find this project with this name in the interface of your online translation solution. Set up this attribute as followed:
  - Attribute type: text 
  - Attribute code: `translationName`
  - Attribute label: `Translation submission name`
  - Attribute group: `Translations`
  - Usable in grid: enabled
* `Translation submitter`: This simple select attribute should contain in its options, all possible contributor names. This attribute allows Julia to associate her name to her translation project. Set up this attribute as followed:
  - Attribute type: simple select  
  - Attribute code: `translationSubmitter`
  - Attribute label: `Translation submitter`
  - Attribute group: `Translations` 
  - Usable in grid: enabled 
* `Translation queued`: This attribute allows Julia to set if a product need to be translated. Set up this attribute as followed:
  - Attribute type: Yes/No (Boolean)
  - Attribute code: `translationQueued`
  - Attribute label: `Translation queued`
  - Attribute group: `Translations`
  - Usable in grid: enabled
* `Translation locales`: This multi select attribute should contains in its `options` all destination locales Julia can use to make her translation requests. These same `locales` must of course also already exist in Akeneo PIM and in your online Translation tool. This attribute allows Julia to choose one or more target locales for her translation project. Set up this attribute as followed:
  - Attribute type: multi select
  - Attribute code: `translationLocales`
  - Attribute label: `Translation locales`
  - Attribute group: `Translations`
  - Usable in grid: enabled
* `Translation due date`: This attribute allows Julia to indicate to translators when she would like her products to be translated. Set up this attribute as followed:
  - Attribute type: date 
  - Attribute code: `translationDate`
  - Attribute label: `Translation due date`
  - Attribute group: `Translations` 
  - Usable in grid: enabled  
* `Translation status`: As this last attribute has a `Read only` property, it can't be modified by Julia. Only your connector can modify this status through API and set the translation project status for each desired locale. This attribute contains these options:    
  |      Code      |     Label      |
  | :------------- | :------------- |
  | INPROGRESS     | In progress    |
  | TRANSLATED     | Translated     |
  | CANCELLED      | Cancelled      |
  Then, set up this attribute as followed:
  - Attribute type: simple select  
  - Attribute code: `translationStatus` 
  - Attribute label: `Translation status`
  - Attribute group: `Translations`  
  - Value per locale: enabled (in order to have a status for each locale)    
  - Read only: enabled (can only be modified via API)   
  - Usable in grid: enabled 

::: tips
To create an attribute:
1. Go to `Settings > Attributes`
2. Click on `CREATE ATTRIBUTE` at the top right corner of the screen
:::
I re-organized this entire section as I didn't understand it at first sight.

```

Reply:
Good idea.